packages/server/src/routes/personal/me.ts

```
@@ -1,56 +1,57 @@
 import { Hono } from "hono";
 import { zValidator } from "@hono/zod-validator";
 import { z } from "zod";
-import { env } from "hono/adapter";
-import { ENV_TYPES } from "../../env/zod";
-import { authMiddleware } from "../../middleware/authentication";
+import { authMiddleware } from "../middleware/authentication";
+import { createRoute, RouteHandler } from "@hono/zod-openapi";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)